packages/mds-provider-processor/src/proc-provider.ts

```diff
@@ -81,16 +82,8 @@ async function processProvider(providerID: UUID, curTime: Timestamp) {
   )
 }
 
-export async function providerAggregator() {
-  const curTime: Timestamp = now()
-  const providersList: UUID[] = config.organization.providers
-  await Promise.all(
-    providersList.map(provider => {
-      try {
-        return processProvider(provider, curTime)
-      } catch (err) {
-        return err
-      }
-    })
-  )
+export async function providerProcessor() {
+  const [{ providers }] = await Promise.all([getConfig(), db.startup()])
```

Comment:
This is good, so we're validating the DB connection is available for R/W before proceeding? I see you've added it to trip-proc, this should be in all of them.